docs/development/postman/tractusx-dtr-access-rule-management-api-v1.0-collection.json

```diff
@@ -0,0 +1,168 @@
+{
+	"info": {
+		"_postman_id": "47f1eefc-bd84-4c6b-9c30-2ff696b1dde0",
+		"name": "tractusx-dtr-access-rule-management-api-v1.0 Copy",
```

Comment:
Is it intentional that the name contains Copy?

Reply:
Done